repos/system_upgrade/el7toel8/actors/checklocalnetworkscripts/libraries/library.py

```diff
@@ -0,0 +1,26 @@
+import os
+from leapp import reporting
+
+
+def generate_report():
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
+
+    # Show documentation url if local scripts of network-scripts installed
+    title = "local scripts of network-scripts detected"
+    summary = "Upgrade will be stopped. Additional steps might be required.\n\n"
```

Comment:
The summary should tell users all they need to know. Reading the summary, it's not helpful in any way. Way is the upgrade stopped? What steps are required? The thing that upgrade is stopped should not even appear in the summary, as that should be obvious from the the point the upgrade is inhibited. Instead of that write summary about the problem. e.g. something like: "The local network-scripts has been detected but these are not supported nor functional on the target system anymore." (my example could be wrong regarding this particular problem, but just giving you a point what is kind of expected to read in summary)

Reply:
Updated. I explained why the upgrade stops and  what steps are required.